src/components/MessageList/utils.ts

```diff
@@ -10,31 +10,52 @@ import type { StreamMessage } from '../../context/ChannelStateContext';
 type ProcessMessagesParams<
   StreamChatGenerics extends DefaultStreamChatGenerics = DefaultStreamChatGenerics
 > = {
-  disableDateSeparator: boolean;
-  hideDeletedMessages: boolean;
-  hideNewMessageSeparator: boolean;
   messages: StreamMessage<StreamChatGenerics>[];
   userId: string;
+  /** Disable date separator in main message list */
+  disableDateSeparator?: boolean;
+  /** Enable date separator in main message list. Has to be accompanied by @param threadList. */
```

Comment:
/** Enable date separator in ~~main~~ THREAD message list. Has to be accompanied by @param threadList. */
I believe this should be changed from 'main message list' to 'thread message list'